src/test/kotlin/org/stellar/walletsdk/helpers/objectFromJsonFile.kt

```diff
@@ -0,0 +1,15 @@
+package org.stellar.walletsdk.helpers
+
+import com.google.gson.Gson
+import java.io.File
+
+fun <T> objectFromJsonFile(filePath: String, t: Class<T>): T {
+  val jsonString =
+    try {
+      File(filePath).readText()
```

Comment:
It is safer to use [try-with-resources](https://www.baeldung.com/kotlin/try-with-resources) to avoid resource leak because File implements auto-closable.